pipeline/psth.py

```diff
@@ -115,26 +271,30 @@ def compute_psth(session_unit_spikes):
 
         return np.array(psth)
```

Comment:
yeah we can remove this `compute_unit_trial_psth()` here.

Reply:
fixed in `c018e278`